src/components/CreateBoard/CreateBoard.jsx

```diff
@@ -0,0 +1,272 @@
+        if (event.target.name === 'category') {
+            for (let i = 0; i < this.state.formGet.category.length; i++) {
+                if (this.state.formGet.category[i].value === event.target.value) {
+                    return this.state.formGet.category[i].key;
+                }
+            }
+        }
+
+        if (event.target.name === 'icon') {
+            for (let i = 0; i < this.state.formGet.icon.length; i++) {
+                if (this.state.formGet.icon[i].value === event.target.value) {
+                    return this.state.formGet.icon[i].key;
+                }
+            }
+        }
+    };
+
+    checkLifeToken = (getToken) => {
+        if (getToken.accessTokenExpiresIn > Date.now()) {
+            return Promise.resolve();
+        } else {
+            return fetch('/auth/update-tokens', {
+                method: 'POST',
+                headers: {
+                    'content-type': 'application/json'
+                },
+                body: JSON.stringify({refreshToken: getToken.refreshToken})
+            })
+                .then(response => response.json())
+                .then(response => {
+                    localStorage.removeItem('TOKEN');
+                    localStorage.setItem('TOKEN', JSON.stringify(response));
+                })
+                .catch(error => {
+                    localStorage.removeItem('TOKEN');
+                    this.setState({errorMessage: error.message});
+                });
+        }
+    };
+
+    onSubmit = (event) => {
+        if (event) {
+            event.preventDefault();
+        }
+
+        let getToken = JSON.parse(localStorage.getItem('TOKEN'));
+        this.checkLifeToken(getToken)
+            .then(() => {
+                return fetch('/board', {
+                    method: 'POST',
+                    headers: {
+                        'content-type': 'application/json',
+                        Authorization: 'Bearer ' + getToken.accessToken,
+                    },
+                    body: JSON.stringify(this.state.formPost)
+                })
+            })
+            .then(response => {
+                this.setState({
+                    responseStatus: response.status
+                });
+                return response.json();
+            })
+            .then(response => {
+                if (this.state.responseStatus === 200) {
+                    customHistory.push('/dashboard');
+                } else {
+                    this.setState({
+                        errorMessage: response.message
+                    });
+                }
+            })
+            .catch(error => {
+                this.setState({
+                    errorMessage: error.message
+                });
+            });
+
+        let key = '';
+        if (!this.props.isEmpty(this.state.formPost.key)) {
```

Comment:
`isEmpty` is a simple function, you don't need to pass it through `props`

Reply:
This function (isEmpty) is used in many components. I thought to transfer it down the tree. need to create a folder where I will store files with the usual functions?

Reply:
done